security/restricting-fiddler-features.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Restricting Fiddler
+description: "Using the Enterprise policy feature to disable specific Fiddler Everywhere features like remote connections, changing ports, cloud services, and others."
+slug: fe-restrict-policies
+publish: true
+position: 40
+---
+
+# Restricting Fiddler
+
+Users of Fiddler Everywhere often work in a secure corporate environment where specific policies are imposed. That means that administrators need to have options to limit or hardcode particular features of the Fiddler Everywhere application. With the Fiddler Everywhere Enterprise tier, administrators can control a set of functionalities through policy management systems on macOS and Windows.
+
+## Restricting Features on macOS
+
+
+Users on macOS with administrative privileges set policies through the [**defaults**](https://macos-defaults.com/) command and by using the `com.progress-telerik.fiddler` domain alongside the following policies:
+
+| Policy          | Description | Value Type |  Value to apply the policy
+|:-----------------|:----------------|:----------------|:-----------------
+| **DisableCloud**   | Disables sharing options  | integer  | 1
+| **DefaultProxyPort**   | Sets default proxy port  | integer  | A number between 0 and 95535
```

Comment:
95535 - probably this is 65535

What is the default proxy port, when is it applied, who will use it? We need to add more information, which will help IT teams to understand if they need to use this key and what will be the impact for the end users.

Reply:
This comment is valid for all the policies